src/librustdoc/clean/types.rs

```diff
@@ -1798,8 +1802,11 @@ impl From<hir::PrimTy> for PrimitiveType {
 
 #[derive(Copy, Clone, Debug)]
 crate enum Visibility {
+    /// `pub`
     Public,
+    /// visibility inherited from parent (e.g. for enum variant fields)
```

Comment:
Private items are more common than enum variants. I don't think we need to mention variant fields, that seems confusing.
```suggestion
    /// Visibility inherited from parent.
    ///
    /// For example, this is the visibility of private items and of enum variants.
```

Reply:
> Private items are more common than enum variants.

I wasn't certain if private items used `Restricted` or `Inherited`, that's why I didn't mention them.

> I don't think we need to mention variant fields, that seems confusing.

Why is that confusing?

Reply:
Variant fields specifically are
```
enum E {
  A {
    x: usize // this one here
  }
}
```
I don't think we need to mention those specifically when we can just say variant.

> I wasn't certain if private items used Restricted or Inherited, that's why I didn't mention them.

They use `Inherited`.